integration_tests/sdk/conftest.py

```diff
@@ -282,22 +282,6 @@ def get_new_flow_name():
         flow_names.append(flow_name)
         return flow_name
 
-    def cleanup_flows():
-        if not pytestconfig.getoption("keep_flows"):
-            for flow_name in flow_names:
-                try:
-                    client.delete_flow(
-                        flow_name=flow_name,
-                        saved_objects_to_delete=client.flow(
-                            flow_name=flow_name
-                        ).list_saved_objects(),
-                    )
-                except Exception as e:
-                    print("Error deleting workflow %s with exception: %s" % (flow_name, e))
-                else:
-                    print("Successfully deleted workflow %s" % flow_name)
-
-    request.addfinalizer(cleanup_flows)
```

Comment:
Actually this is huge. The entire point of the `flow_name()` fixture was to register each new flow  at the beginning of each test, and then tear it down at the end. Now that we're using your `delete_all_flows()` method, this fixture really should come useless. Can you make a task for this and tag it on this fixture so we can deprecate?

Reply:
I actually think it's still very useful to avoid duplicated names across test cases?

Reply:
yeah but it doesn't need to be a fixture anymore, we can just use a helper which is a lot easier (eg. publish_flow_test() can take an optional flow_name, and if it isn't set, it'll generate a random flow name for you).

Reply:
Makes sense. Let's do this separately as it seems to require us change every single test case https://linear.app/aqueducthq/issue/ENG-2840/remove-flow-name-fixture